.circleci/config.yml

```
@@ -2,14 +2,15 @@
 version: 2.1
 
 orbs:
-  python: circleci/python@1.2.1
+  python: circleci/python@3.0.0
 
 jobs:
   build_docs:
     executor: python/default
     steps:
       - checkout
       - python/install-packages:
+          args: --upgrade --upgrade-strategy=eager
```

```suggestion
          args: --upgrade --upgrade-strategy eager
```
Perhaps this will work? The [pip docs suggest](https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-upgrade-strategy) there doesn't need to be a `=`.